host_provider/templates/k8s/yamls/statefulset.yaml

```diff
@@ -44,65 +44,29 @@ spec:
             - containerPort: {{ CONTAINER_PORT }}
           env:
             - name: MONGO_INITDB_ROOT_USERNAME
-              valueFrom:
-                secretKeyRef:
-                  name: mongodb-users-secrets
-                  key: MONGODB_ADMIN_USER
+              value: {{ INIT_USER }}
             - name: MONGO_INITDB_ROOT_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: mongodb-users-secrets
-                  key: MONGODB_ADMIN_PASSWORD
+              value: {{ INIT_PASSWORD }}
             - name: MONGODB_ADMINDBAAS_USER
-              valueFrom:
-                secretKeyRef:
-                  name: mongodb-users-secrets
-                  key: MONGODB_ADMINDBAAS_USER
+              value: admindbaas
             - name: MONGODB_ADMINDBAAS_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: mongodb-users-secrets
-                  key: MONGODB_ADMINDBAAS_PASSWORD
+              value: a
```

Comment:
This password `a` sounds weird, maybe we could use something like `startup_pwd` or `tmp_pwd`